src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskLevelCollectionViewCell.swift

```diff
@@ -25,7 +25,7 @@ protocol RiskLevelCollectionViewCellDelegate: AnyObject {
 final class RiskLevelCollectionViewCell: HomeCardCollectionViewCell {
 	// MARK: Properties
 
-	var delegate: RiskLevelCollectionViewCellDelegate?
+	weak var delegate: RiskLevelCollectionViewCellDelegate?
```

Comment:
Hi @aleksandr-tikhonov ,
Did you omit the `weak` on purpose?
Thanks.

Reply:
I think no, probably I did it when I was debugging something and forgot to revert it.